### PR TITLE
Test: Streaming with char encoding

### PR DESCRIPTION
**Pull request by v-kigos**

### CODE REVIEW

File: test/sqlsrv/srv_013_sqlsrv_get_field.phpt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+sqlsrv_get_field() using SQLSRV_PHPTYPE_STREAM(SQLSRV_ENC_CHAR)
+--SKIPIF--
+--FILE--
+<?php
+
+require_once("autonomous_setup.php");
+
+// Connect
+$conn = sqlsrv_connect( $serverName, $connectionInfo);
+if( !$conn ) { die( print_r( sqlsrv_errors(), true)); }
```
Comment (yitam) [inline test/sqlsrv/srv_013_sqlsrv_get_field.phpt]: Would you consider creating a helper method for error handling (die(print_r(sqlsrv_errors...))?

File: test/sqlsrv/srv_013_sqlsrv_get_field.phpt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+// Create database
+sqlsrv_query($conn,"CREATE DATABASE ". $dbName) ?: die();
```
Comment (yitam) [inline test/sqlsrv/srv_013_sqlsrv_get_field.phpt]: again, is a new database necessary?

File: test/sqlsrv/srv_013_sqlsrv_get_field.phpt [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+// Create table
+$query = "CREATE TABLE ".$tableName." (ID NVARCHAR(10))";
+$stmt = sqlsrv_query($conn, $query);
+
+// Insert data
+$query = "INSERT INTO ".$tableName." VALUES ('1998.1'),('-2004.2436'),('4.2 EUR')";
+$stmt = sqlsrv_query($conn, $query) ?: die( print_r( sqlsrv_errors(), true) );
+
+// Fetch data
+$query = "SELECT * FROM ".$tableName;
```
Comment (yitam) [inline test/sqlsrv/srv_013_sqlsrv_get_field.phpt]: Just a suggestion: $query = "SELECT * FROM $tableName";

File: test/sqlsrv/srv_013_sqlsrv_get_field.phpt [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+// Connect
+$conn = sqlsrv_connect( $serverName, $connectionInfo);
+if( !$conn ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Create database
+sqlsrv_query($conn,"CREATE DATABASE ". $dbName) ?: die();
+
+// Create table
+$query = "CREATE TABLE ".$tableName." (ID NVARCHAR(10))";
```
Comment (yitam) [inline test/sqlsrv/srv_013_sqlsrv_get_field.phpt]: Is there any reason for NVARCHAR when no unicode character is used?

File: test/sqlsrv/srv_013_sqlsrv_get_field.phpt [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+// Insert data
+$query = "INSERT INTO ".$tableName." VALUES ('1998.1'),('-2004.2436'),('4.2 EUR')";
+$stmt = sqlsrv_query($conn, $query) ?: die( print_r( sqlsrv_errors(), true) );
+
+// Fetch data
+$query = "SELECT * FROM ".$tableName;
+$stmt = sqlsrv_query( $conn, $query ) ?: die( print_r( sqlsrv_errors(), true) );
+
+while(sqlsrv_fetch($stmt)) {
+	$field = sqlsrv_get_field($stmt, 0, SQLSRV_PHPTYPE_STREAM(SQLSRV_ENC_CHAR));
```
Comment (yitam) [inline test/sqlsrv/srv_013_sqlsrv_get_field.phpt]: Is Unicode is used, we can't use SQLSRV_ENC_CHAR right?

File: test/sqlsrv/srv_048_stream_nvarchar.phpt [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+if( !$stmt ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Fetch
+$sql = "SELECT * FROM #Table";
+$stmt = sqlsrv_query($conn, $sql);
+if( !$stmt ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Compare output
+$row = sqlsrv_fetch_array($stmt);
+echo ($row['c1'] == $data) ? "True\n" : "False\n";
```
Comment (yitam) [inline test/sqlsrv/srv_048_stream_nvarchar.phpt]: No trimming necessary? Nice. :) Just curious, have you tried using '===' for comparison?

File: test/sqlsrv/srv_049_stream_nvarchar_utf8.phpt [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+if( !$stmt ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Fetch
+$sql = "SELECT * FROM #Table";
+$stmt = sqlsrv_query($conn, $sql);
+if( !$stmt ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Compare output
+$row = sqlsrv_fetch_array($stmt);
+echo ($row['c1'] == $data) ? "True\n" : "False\n";
```
Comment (yitam) [inline test/sqlsrv/srv_049_stream_nvarchar_utf8.phpt]: The main diff of this test comparing to the one above is UTF-8 charset right? Are both files saved in the same encoding?

File: test/sqlsrv/srv_013_sqlsrv_get_field.phpt (hunk 1 above)
Comment (v-kigos) [inline test/sqlsrv/srv_013_sqlsrv_get_field.phpt]: done

**Pull request merged**